contracts/factory/FactoryStorage.sol

```diff
@@ -11,10 +11,11 @@ contract FactoryStorage is AccessControl {
 
     uint16 public feeBPS;
     address public feeRecipient;
-    address public vaultPackage;
     address[] public vaults;
-    mapping(address => address) public vaultCreators;
+    address[] public vaultPackages;
 
     /// @notice Initialized state of the factory.
     bool public initialized;
+
+    mapping(address => address) public vaultCreators;
```

Comment:
This was implemented to be used in the future on the subgraph

Reply:
In this case, it's better to add this data to the emitted event instead of keeping it in storage. Let's remove this mapping then and update `VaultPackageAdded` event